README.md

```diff
@@ -27,6 +27,52 @@ expected syntax (and semantics) for all such values is that of
 [Perl regular expressions](https://perldoc.perl.org/perlre.html#Regular-Expressions).
 
 
+### Accounts
+
+ledger2beancount will convert ledger account declaration to beancount
+`open` statements using the `account_open_date` variable as the opening
+date.  The `note` is used as the `description`.
+
+You can also add account mappings to `account_map` for automatic
+conversion of account names.
+
+
+### Commodities
+
+Like accounts, ledger2beancount will convert ledger commodity
+declarations to beancount.  The `note` is converted to `name`.
+
+ledger2beancount will automatically convert all commodities to uppercase
+and strip quoted commodities.  If you require a mapping between ledger
+and beancount commodities, you can use `commodity_map`.
+
+Commodity symbols (like $, € and £) are not supported.
+
+
+### Flags
+
+ledger2beancount supports both transaction flags and account flags.
+
+
+### Dates
+
+Only dates in the format YYYY-MM-DD (ISO 8601) are currently supported.
+
+
+### Auxiliary dates
+
+Beancount currently doesn't support auxiliary dates so these are stored
+as metadata according to the `altdate_tag` variable.  Unset the variable
+if you don't want auxiliary dates to be stored as metadata.  Account
+and posting-level auxiliary dates are supported.
+
+
+### Codes
```

Comment:
This heading should probably be Transaction codes.
And it would be nice to point to the relevant description in the Ledger documentation, … if only there were a relevant target (I have looked, but haven't found one).

Reply:
It's the section right after aux dates :)

https://www.ledger-cli.org/3.0/doc/ledger3.html#Codes